syntax ="proto3";
//import "google/protobuf/descriptor.proto";
//import "google/protobuf/repeated_filed.proto";

package PB;

message UserInfo 
{
	int32		UserID			= 1;
	string		UserName		= 2;
	string		EnglishName		= 3;
	string		Email			= 4;
	int32 		Level			= 5;		// 职级 
	string		WeChatID		= 6;
	string		PhoneNum		= 7;
	int32 		DepartmentID	= 8;
	string		DepartmentName	= 9;
	int32		FocusLevel		= 10;
	bool		IsCC			= 11;
}

message GroupInfo 
{
	int32	GroupID					= 1;
	string	GroupName				= 2;
	bool	IsCC					= 3;
	int32	FocusLevel				= 4;
	repeated UserInfo listUserInfo  = 5;
}

message BizInfo 
{
	int32	 BizID						= 1;
	string	 BizName					= 2;
	int32	 AffDepartment				= 3;
	string	 Description				= 4;
	repeated UserInfo	listUserInfo	= 5;
	repeated GroupInfo	listGroupInfo	= 6;
}


